cmd/goexports/goexports.go

```diff
@@ -212,15 +222,17 @@ func genContent(dest, pkgName string) ([]byte, error) {
 		buildTags += ",!windows,!nacl,!plan9"
 	}
 
-	b := &bytes.Buffer{}
+	b := new(bytes.Buffer)
 	data := map[string]interface{}{
-		"Dest":      dest,
-		"Imports":   imports,
-		"PkgName":   pkgName,
-		"Val":       val,
-		"Typ":       typ,
-		"Wrap":      wrap,
-		"BuildTags": buildTags,
+		"Dest":        dest,
+		"Imports":     imports,
+		"PkgName":     pkgName,
+		"Val":         val,
+		"Typ":         typ,
+		"Wrap":        wrap,
+		"BuildTags":   buildTags,
+		"License":     license,
+		"BlankImport": len(val)+len(typ)+len(wrap) == 0,
```

Comment:
It's not related to the license support, so could you remove those changes from this PR, thanks.

Reply:
done